modules/swagger-codegen/src/main/resources/go-server/README.mustache

```diff
@@ -0,0 +1,21 @@
+# Swagger generated server
+
+## Overview
+This server was generated by the [swagger-codegen](https://github.com/swagger-api/swagger-codegen) project.  By using the [OpenAPI-Spec](https://github.com/OAI/OpenAPI-Specification) from a remote server, you can easily generate a server stub.  This is an example of building a node.js server.
```

Comment:
README wasn't updated. This line references a node.js server. Also, instructions for running the server are missing.